packages/css/src/components/accordion/accordion.template.ts

```diff
@@ -0,0 +1,60 @@
+import { Accordion } from '@dso-toolkit/sources';
+
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+import { accordionHandleHeading } from './accordion-handle-heading.template';
+
+export function accordionTemplate({ variant, sections, reverseAlign }: Accordion) {
+  return html`
+    <div class="dso-accordion ${classMap({
+      [`${variant}`]: !!variant,
+      'dso-align-viewer': !!reverseAlign
```

Comment:
Waarom is de naamgeving van deze twee properties zo verschillend?

Reply:
Mee eens, ik stel voor om de class te hernoemen naar `dso-accordion-reverse-view`